### PR TITLE
Make minimum and maximum aliases for clamp_max and clamp_min

### PR DESCRIPTION
**Pull request by ngimel**
Also, fix nan propagation for clamp_max and clamp_min

### CODE REVIEW

File: torch/csrc/lazy/core/shape_inference.cpp [hunk 1]
```diff
@@ -554,6 +554,14 @@ std::vector<Shape> compute_shape_clamp_min(const at::Tensor & self, const at::Sc
   return {Shape(self.scalar_type(), self.sizes().vec())};
 }
 
+std::vector<Shape> compute_shape_minimum(const at::Tensor & self, const at::Tensor & other) {
```
Comment (ngimel) [inline torch/csrc/lazy/core/shape_inference.cpp:557]: cc @wconstab. 
I'm making minimum/maximum alias of clamp_min/clamp_max so they are not structured anymore and require shape functions. These are the wrong shape functions (should broadcast self and other), so what would be the right shape functions, or how can I let lazy tensor know that minimum and maximum should use clamp_min's shape function?

Comment (bdhirsh) [reply torch/csrc/lazy/core/shape_inference.cpp:557]: Will might have a better suggestion, but clamp_min/clamp_max just use meta tensors in LTC today, so you could copy what they do:

```
auto out_meta = at::meta::clamp_min(self, other);
return {Shape(out_meta.scalar_type(), out_meta.sizes().vec())};
```

Comment (bdhirsh) [reply torch/csrc/lazy/core/shape_inference.cpp:557]: oh or maybe an even better option: now that `minimum` and `maximum` are `CompositeExplicitAutograd`, LTC should get them for free (and they'll decompose into clamp_min/clamp_max).

So you can just remove `minimum` and `maximum` from `ts_native_functions.yaml` and let the decomp do the work: https://github.com/pytorch/pytorch/blob/8f5cdc6d5d98f4dfc6d9ef27f43a52957bf1abb9/aten/src/ATen/native/ts_native_functions.yaml#L85

**Comment by ngimel** [comment]
Closing in favor of #77306

**Pull request closed without merging**